lib/wallaby/feature_case/ecto_sql/sandbox.ex

```diff
@@ -0,0 +1,8 @@
+unless Code.ensure_loaded?(Ecto.Adapters.SQL.Sandbox) do
+  defmodule Ecto.Adapters.SQL.Sandbox do
+    @moduledoc false
+
+    def checkout(_repo), do: nil
+    def mode(_repo, _mode), do: nil
+  end
+end
```

Comment:
@aaronrenner I think this should alleviate the following concerns. What do you think?

>Would a FeatureCase be usable if I'm not using ecto? I see it adds a dependency on ecto_sql and phoenix_ecto, which may not be needed if someone is using wallaby for smoke testing or scraping non-phoenix websites.

Reply:
Instead of defining fake replacement modules, could there be a flag in the feature case that disables the Ecto sandbox integration?

Reply:
What are you imagining?

I'd like to clarify that if the user doesn't configure their `otp_app`, the ecto stuff has no effect on their tests. 

The feature case still makes automatic screenshots more useful (taking them on any test failure and providing a name that is based on the test's name) and making it easier to create a feature out of the box (the user doesn't have to create their own feature case).